docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md

```diff
@@ -95,17 +95,26 @@ For example, if you want to send a message in a Microsoft Teams channel, choose
+|      Topic      |                                                                Create a new chat                                                                |    No    |     string      |                                                                                    Topic of chat                                                                                     |
+|     Members     |                                                                Create a new chat                                                                |   Yes    | FEEL expression |                                                               See [members property](#members-property) to learn more.                                                               |
+|       Top       |                                                              List messages in chat                                                              |    No    |     numbers     |                                                     Controls the number of items per response; maximum allowed top value is 50.                                                      |
+|    Order by     |                                                              List messages in chat                                                              |   Yes    |    dropdown     |                                                              Can order by 'lastModifiedDateTime' and 'createdDateTime'.                                                              |
+| Expand response |                                                                 Get chat by id                                                                  |   Yes    |    dropdown     |                                                                                        Choose                                                                                        |
+|     Filter      |                                                              List messages in chat                                                              |    No    |     string      | Sets the date range filter for the lastModifiedDateTime and createdDateTime properties. [Learn more about filtering](https://learn.microsoft.com/en-us/graph/filter-query-parameter) |
+|   Message ID    |                                                               Get message in chat                                                               |   Yes    |     string      |                                                                           Microsoft teams chat message id                                                                            |
+
+##### Expand response
+
+You can expand response for method **Get chat by ID**. For this in **Expand response** property you can :
```

Comment:
I would suggest "For method  **Get chat by ID**, you can get more information in the response by using the dropdown property  **Expand response**. You can choose one of the following values: "